tests/pipeline/spatialtranscriptomics.nf.test

```diff
@@ -0,0 +1,27 @@
+nextflow_pipeline {
+    name "Test Spatialtranscriptomics QC workflow"
```

Comment:
```suggestion
    name "Test Workflow main.nf"
```

Reply:
would recommend leaving the name default that comes with generate as we would like that go into template.